### PR TITLE
Add wrappers for brightness-related functions

### PR DESCRIPTION
**Pull request by exelix11**

### CODE REVIEW

File: nx/source/services/lbl.c [hunk 1]
```diff
@@ -70,3 +70,147 @@ Result lblSwitchBacklightOn(u64 fade_time) {
...
+
+        rc = resp->result;
+        if (R_SUCCEEDED(rc))
+            *out_value = resp->brightness;
+    }
+
+    return rc;
+}
+
+static Result _lblSetAutoBrightnessControl(u32 cmd_id) {
```
Comment (yellows8) [inline nx/source/services/lbl.c]: Rename to _lblCmdNoIO and move it right after lblExit.

File: nx/source/services/lbl.c [hunk 2]
```diff
@@ -29,6 +29,38 @@ void lblExit(void) {
     }
 }
 
+static Result _lblCmdNoIO(u32 cmd_id) {
```
Comment (yellows8) [inline nx/source/services/lbl.c]: u64 cmd_id

Comment (exelix11) [reply nx/source/services/lbl.c]: At line 64 too ?
https://github.com/switchbrew/libnx/pull/246/files/15ae541137274156fa6ad0ce8c4e9da9674347e0#diff-5d1e5f536b1977d6163897b420db6572R64

Comment (yellows8) [reply nx/source/services/lbl.c]: yes

File: nx/include/switch/services/lbl.h [hunk 3]
```diff
@@ -12,3 +12,13 @@ void lblExit(void);
 
 Result lblSwitchBacklightOn(u64 fade_time);
 Result lblSwitchBacklightOff(u64 fade_time);
+
+/**
+ * @note The brightness goes from 0 to 1.0
```
Comment (yellows8) [inline nx/include/switch/services/lbl.h]: period

**Comment by yellows8** [comment]
Probably should update "@ author" FWIW.

**Pull request merged**